support/cas-server-support-oauth/src/test/java/org/apereo/cas/support/oauth/util/OAuth20UtilsTests.java

```diff
@@ -81,6 +83,17 @@ public void verifyGrants() {
             OAuth20GrantTypes.PASSWORD.getType(), new OAuthRegisteredService()));
     }
 
+    @Test
+    public void verifyCheckCallbackValid() {
+        val registeredService = mock(RegisteredService.class);
```

Comment:
Do not use a mock please. Use a proper `val registeredService = new OAuthRegisteredService();
`